task2_Gal_Malka/include/ChessBoard.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+#include "ChessPiece.h"
+#include "Pawn.h"
+#include "Knight.h"
+#include "King.h"
+#include "Queen.h"
+#include "Rook.h"
+#include "Bishop.h"
+
+const int BOARD_SIZE = 8;
```

Comment:
For consts like this use `constexpr`
Check it out: https://learn.microsoft.com/en-us/cpp/cpp/constexpr-cpp?view=msvc-170#constexpr-variables